CMakeLists.txt

```
@@ -8,6 +8,7 @@ project(simulator)
 option(BUILD_PROJECT "Build tests" ON)
 
 if (BUILD_PROJECT)
+    include_directories(${CMAKE_SOURCE_DIR}/spdlog/include/)
```

I think it would be better to include our logger always:
```suggestion
include_directories(${CMAKE_SOURCE_DIR}/spdlog/include/)
if (BUILD_PROJECT)
```

---

I agree with you.